handbook/direction/code-graph/search/index.md

```diff
@@ -1,36 +1,150 @@
-# Search goals and priorities
+# Search direction
 
-This page documents the current goals and priorities of the [Search team](../../../engineering/code-graph/search/index.md). See also our [completed goals](../../../engineering/code-graph/search/goals_completed.md).
```

Comment:
@benvenker removing this line was the only link to `goals_completed.md` in the handbook, which is why you're getting the error now that the page is disconnected.

Reply:
@jyavorska talked with @yinquanteo and I'm going to keep it because there are other completed goals pages for the Code Graph team that didn't break the build and so must be linked other places. i don't want to wipe a page someone may be using...